nvm.sh

```diff
@@ -1181,17 +1180,17 @@ nvm_ls_remote_index_tab() {
     setopt shwordsplit
   fi
   local VERSION_LIST
-  VERSION_LIST="$(nvm_download -L -s "${MIRROR}/index.tab" -o - \
-    | command sed "
+  VERSION_LIST="$(nvm_download -L -s "${MIRROR}/index.tab" -o - |
+    command sed "
         1d;
         s/^/${PREFIX}/;
-      " \
+      "
   )"
   local LTS_ALIAS
   local LTS_VERSION
   command mkdir -p "$(nvm_alias_path)/lts"
-  nvm_echo "${VERSION_LIST}" \
-    | command awk '{
```

Comment:
this change is worse; starting the line with `|` makes it more clear that it's a continuation.

Reply:
Isn't this a pipe?

Reply:
i mean, a continuation of the command that started on line 1193

Reply:
Sure. Fixed.